packages/MorphicAPIExplorer-Core.package/MEXToolExplorationView.class/instance/evaluateCode..st

```diff
@@ -3,6 +3,6 @@ evaluateCode: aCode
 
 	| evaluatedCode |
```

Comment:
Empty line after local vars and comment.
```suggestion
	| evaluatedCode |
	
```

Reply:
I think that's still not in the coding standards and most of the code actually enforces no empty line